src/infra/archive-stable-version-installers.md

```diff
@@ -0,0 +1,22 @@
+# Archive of Rust Stable Standalone Installers
+
+The official Rust standalone installers contain a single release of Rust, and
```

Comment:
Can we add some explicit disclaimer here about lack of maintenance? Something like:

**Note: The Rust project only supports the latest stable release with security patches. Generally speaking these archives should not be used without some extra mechanisms to provide for patching.**

Reply:
Added.